### PR TITLE
[OPPRO-316] Support StddevSamp on Velox backend

### PR DESCRIPTION
**Pull request by rui-mo**
## What changes were proposed in this pull request?

This PR supports StddevSamp on Velox backend.

## How was this patch tested?

Verified on Jenkins.

### CODE REVIEW

**Review by zzcclp — APPROVED** [review]
LGTM

File: backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala [hunk 1]
```diff
@@ -365,4 +365,22 @@ class TestOperator extends WholeStageTransformerSuite {
       }
     }
   }
+
+  test("stddev_samp") {
+    withSQLConf("spark.sql.adaptive.enabled" -> "false") {
```
Comment (JkSelf) [inline backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala:370]: Whether Stddev not support AQE?

Comment (rui-mo) [reply backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala:370]: This is due to an unresolved common issue I found in plan check. In the unit tests, if AQE is enabled, in some cases `transformer` cannot be found through the executed plan.  So in this file some unit tests were marked as AQE disabled for now.

**Review by JkSelf — APPROVED** [review]
LGTM.

**Comment by rui-mo** [comment]
Thanks for your review!

**Pull request merged**